include/rootston/view.h

```diff
@@ -92,6 +107,13 @@ struct roots_view {
 	int border_width;
 	int titlebar_height;
 
+	bool special; // marks special "system" feature set
```

Comment:
gross

Reply:
ugly - this code will go when panel interface is reconstructed, so I went crazy here.